voici/tree_exporter.py

```diff
@@ -71,15 +95,21 @@ def __init__(
         voici_configuration: VoilaConfiguration,
         **kwargs,
     ):
-        self.jinja2_env = jinja2_env
+        self.jinja2_env = self._environment_cached = jinja2_env
```

Comment:
```suggestion
        self.jinja2_env = jinja2_env
```

?

Reply:
Needs the `self._environment_cached` in order to make `self._init_resources()` works